### DISCUSSION TITLE
Cata race/class combos

### DISCUSSION THREAD

**Discussion by jtedmons** [chromiecraft/chromiecraft#3329] [Suggestions]
I know I know ew Cata, but I think opening up more combinations was one of the best parts.  Not sure if it's possible, but obviously no worgen or goblins.  I would even say no need to add in the class npcs to the starting areas, unless that's easy.  Could literally have them follow the same path as the existing classes for combinations  ie tauren pallies do their class quests in Silvermoon, troll druids in Thunder Bluff, etc.  Obviously with the custom portals in each major city it wouldn't be a huge inconvenience.  The combos could even just start in that area again troll druids start in mulgore etc.  It would open up a lot of options for roleplay, transmog, and possibly new players/people leveling more alts.

**Comment by Nyeriah** [comment]
Hey,

Thanks for the suggestion. This is a lovely idea, however, it's not possible without severe client modifications, and that is something unfortunatelly we can't do.

For example, for the classes to even show up in the creation screen it requires client changes, same to make them display properly in-game.

Regards.

**Reply by jtedmons** [reply]
Ahh that's too bad.  What about race trasmogs?  Just changing appearance leaving racials alone?

**Reply by Hatepoem** [reply]
> Hey,
> 
> Thanks for the suggestion. This is a lovely idea, however, it's not possible without severe client modifications, and that is something unfortunatelly we can't do.
> 
> For example, for the classes to even show up in the creation screen it requires client changes, same to make them display properly in-game.
> 
> Regards.

This is something that I am working on with my local instance of AzerothCore. Yes, it does require client modifications. From what I understand it's merely an issue of providing a patch MPQ file.

It could be possible to make custom race-class combinations to be unlocked with the Chromie Shop. That is modify the database records with a Web interface, instead of relying on the core and client functionality solely. 

+++ Unfortunately it's not as simple as modifying a class field in a character entry. Specifically the class trainers are no longer available to such a character, presenting empty lists of skills. Still I think it's possible to make work.

If I were to provide a workable solution, an AzerothCore module, will it be considered for deployment?

**Reply by Nyeriah** [reply]
(Not a staff member anymore) but still sadly it is very unlikely, as 1) providing the patches is really bothersome as everyone needs to download additional files 2) modifying the client is an illegal practice and thus not condoned by AzerothCore
